packages/lumx-react/src/components/alert-dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,185 @@
+    const titleId = `${uniqueId}-title`;
+    const descriptionId = `${uniqueId}-description`;
+
+    // If content is a string, set in a paragraph.
+    const DescriptionElement = typeof children === 'string' ? 'p' : 'div';
+
+    return (
+        <Dialog
+            ref={ref}
+            focusElement={cancelProps ? cancelButtonRef : confirmationButtonRef}
+            size={size}
+            dialogProps={{
+                id: uniqueId,
+                role: 'alertdialog',
+                'aria-labelledby': titleId,
+                'aria-describedby': descriptionId,
+                ...dialogProps,
+            }}
+            className={classNames(
+                className,
+                handleBasicClasses({
+                    kind,
+                    prefix: CLASSNAME,
+                }),
+            )}
+            {...forwardedProps}
+        >
+            <header>
+                <Toolbar
+                    label={
+                        <FlexBox orientation={Orientation.horizontal} hAlign={Alignment.center}>
+                            {icon && (
+                                <Icon
+                                    className="lumx-spacing-margin-right-big"
+                                    icon={icon}
+                                    size={Size.s}
+                                    color={color}
+                                />
+                            )}
+                            <h2 id={titleId} className="lumx-typography-title">
+                                {title}
+                            </h2>
+                        </FlexBox>
+                    }
+                />
+            </header>
+
+            {children && (
+                <DescriptionElement
+                    id={descriptionId}
+                    className="lumx-typography-body2 lumx-spacing-padding-vertical-big lumx-spacing-padding-horizontal-huge"
+                >
+                    {children}
+                </DescriptionElement>
+            )}
+
+            <footer>
+                <Toolbar
+                    label={
+                        <FlexBox vAlign={Alignment.right}>
+                            {cancelProps && (
+                                <Button ref={cancelButtonRef} emphasis={Emphasis.medium} onClick={cancelProps.onClick}>
+                                    {cancelProps.label}
+                                </Button>
+                            )}
+                            <Button
+                                ref={confirmationButtonRef}
+                                color={color}
+                                className="lumx-spacing-margin-left-regular"
+                                onClick={confirmProps.onClick}
+                            >
+                                {confirmProps.label}
+                            </Button>
+                        </FlexBox>
+                    }
+                />
```

Comment:
```suggestion
                <Toolbar
                    after={
                        <>
                            {cancelProps && (
                                <Button ref={cancelButtonRef} emphasis={Emphasis.medium} onClick={cancelProps.onClick}>
                                    {cancelProps.label}
                                </Button>
                            )}
                            <Button
                                ref={confirmationButtonRef}
                                color={color}
                                className="lumx-spacing-margin-left-regular"
                                onClick={confirmProps.onClick}
                            >
                                {confirmProps.label}
                            </Button>
                        </>
                    }
                />
```

Reply:
Unfortunately, that won't work :( 
If we use the `after` prop, the Toolbar sets a padding of 16px instead of 24...